app/jobs/fetch_job.rb

```diff
@@ -17,35 +15,24 @@ class FetchJob < ApplicationJob
   def perform(fetch_month)
     @fetch_month = fetch_month
     running_status(fetch_month)
-    FileUtils.mkdir_p(fetch_month.crawl_directory)
     fetch_warcs
     return success_status(fetch_month) unless web_archives?(fetch_month.crawl_directory)
 
     cocina_obj = register(request_params)
     start_workflow(cocina_obj.externalIdentifier, cocina_obj.version)
     success_status(fetch_month, druid: cocina_obj.externalIdentifier)
+  rescue RuntimeError => e
```

Comment:
There are a bunch of exception classes that are children of `StandardError` and not children of `RuntimeError` that this will not catch, I don't think. _E.g._, `ArgumentError`, `IOError`, and `NoMethodError`. (You can see a snapshot of the Ruby exception hierarchy here if helpful: https://www.honeybadger.io/blog/understanding-the-ruby-exception-hierarchy/) Intended or nah?